cpp/src/ray/test/cluster/cluster_mode_test.cc

```diff
@@ -408,6 +405,17 @@ TEST(RayClusterModeTest, TaskWithPlacementGroup) {
   ray::RemovePlacementGroup(placement_group.GetID());
 }
 
+TEST(RayClusterModeTest, ObjectRefArgsTest) {
+  auto obj = ray::Put(std::string("aaa"));
+  auto r = ray::Task(GetVal).Remote(obj);
+  EXPECT_EQ(*r.Get(), "aaa");
+
+  auto obj1 = ray::Put(1);
+  auto obj2 = ray::Put(2);
+  auto r1 = ray::Task(Add).Remote(obj1, obj2);
+  EXPECT_EQ(*r1.Get(), 3);
+}
+
```

Comment:
Please add test cases about `ray::Get(std::vector<ray::ObjectRef>)` and `ray::Task` with a `std::vector<ray::ObjectRef>` argument.